config/application.rb

```diff
@@ -29,6 +30,10 @@ class Application < Rails::Application
       allow do
         origins "*"
         resource "/graphql", headers: :any, methods: [:post]
+
+        if Rails.env.development?
+          resource "/browse/*", headers: :any, methods: [:options, :get, :post, :patch]
```

Comment:
If this is just in development do we have to add CORS for this path to nginx?

Reply:
Add comment: This makes development work when you're running the react front-end on a different port from the rails server. Permit the front-end to access the browse everything controllers.